content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
```

Comment:
TODO once https://github.com/istio/istio.io/pull/4928 is merged, link to that page

Reply:
@howardjohn please change the PR content to remove the "this is not safe to merge" language and otherwise tidy up the github message.

Reply:
its merged, issues have been addressed. Link is https://preliminary.istio.io/docs/ops/traffic-management/protocol-selection/

Reply:
@howardjohn thanks will address.

Reply:
@howardjohn this is resolved in the latest set of commits.  If you agree, please click `Resolve conversation`.